ui/lib/pki/addon/routes/certificates/index.js

```diff
@@ -11,8 +11,9 @@ import { getCliMessage } from 'pki/routes/overview';
 
 @withConfig()
 export default class PkiCertificatesIndexRoute extends Route {
-  @service store;
+  @service pagination;
   @service secretMountPath;
+  @service store; // used by @withConfig decorator
```

Comment:
thanks for the comment

Reply:
I wonder if the `withConfig` decorator should inject the store instead if it needs it?

Reply:
yep - I almost made that change and can't remember why I didn't 🤔 I think to keep the scope small...because I wanted to change both config decorators and cleanup unneeded store injections